### PR TITLE
fix: remove *big.Int from API, use string to send bigints

### PR DESCRIPTION
**Pull request by MaxMoskalenko**

### CODE REVIEW

**Comment by dimast-x** [comment]
We used big its to enforce client to provide values in bigInt format without decimal points where is was actually needed (in blockchain-specific endpoints for operations with channels). I am concerned that this implementation with decimals would allow clients to submit values like 28262.82, which would break the system. Also it becomes not clear for a client which format of number should they provide. With big ints it was straightforward.

What issue this update actually fixes?

**Comment by MaxMoskalenko** [comment]
No, it's not SDK problem. Because the main pain point is JSON marshalling/unmarshalling on `http` side. That's very dangerous  to send numbers that are bigger than Number.MAX_SAFE_INTEGER (https://community.n8n.io/t/handling-large-numbers-in-http-requests-precision-loss-issue/50305)

But I understand your, @dimast-x, concern, and the best approach that I could suggest is to add regex binding pattern to accept only `^[0-9]+$` (numeric) values in gin

File: clearnode/channel_service.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
```
[the remark this answers is not in the document]
Comment (dimast-x) [reply clearnode/channel_service.go:9]: Oh, understood

**Comment by alessio** [comment]
> But I understand your, @dimast-x, concern, and the best approach that I could suggest is to add regex binding pattern to accept only `^[0-9]+$` (numeric) values in gin

I agree. In addition, considering how sensitive this whole thing is, I would take as much time as necessary to harden input validation as much as possible as well as write enough tests for the trickiest edge cases.

File: clearnode/channel_service.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (alessio) [reply clearnode/channel_service.go:9]: I am not entirely sure but you might find something of interest [here](https://github.com/cosmos/cosmos-sdk/tree/main/math).

**Review by dpatsora — APPROVED** [review]
